testData.json

```diff
@@ -1 +1 @@
-{"tags":[{"_key":"0","published":true,"type":"tag","text":"plants"},{"_key":"1","published":true,"type":"tag","text":"animals"},{"_key":"2","published":true,"type":"tag","text":"placenames"},{"_key":"3","published":true,"type":"tag","text":"songs"},{"_key":"4","published":true,"type":"tag","text":"legends"}],"terms":[{"_key":"1","published":false,"type":"term","term":"Chil-term-1","termEnglish":"Engl-term-1","contributorId":"John Doe"},{"_key":"2","published":false,"type":"term","term":"Chil-term-2","termEnglish":"Engl-term-2","contributorId":"John Doe"},{"_key":"3","published":false,"type":"term","term":"Chil-term-no-english","contributorId":"Jane Deer"}],"vocabulary_lists":[{"_key":"vocabulary-list-id-1","published":false,"type":"vocabularyList","name":"test VL 1 chil","nameEnglish":"test VL 1 engl","entries":[{"termId":"1","variableValues":{"person":"11"}},{"termId":"2","variableValues":{"person":"21"}}],"variables":[{"name":"person","type":"dropbox","validValues":[{"display":"I","value":"11"},{"display":"We","value":"21"}]}]},{"_key":"vocabulary-list-id-2","published":false,"type":"vocabularyList","name":"test VL 2 CHIL- no engl name","entries":[{"termId":"2","variableValues":{"person":"23"}}],"variables":[]}]}
```

Comment:
Gitignore this file

Reply:
Done. I moved the write location to the same directory of the test as well.